examples/analysis/tabs/create_tables.jl

```diff
@@ -0,0 +1,24 @@
+# This scripts calls all scripts to create tables for comparing forecasts of the abm and the baseline models (AR, ARX, VAR, VARX)
+import BeforeIT as Bit
+using LaTeXStrings, CSV, HDF5, MAT
+
+# Clear old files
+foreach(rm, filter(endswith(".h5"), readdir("./examples/analysis/tabs/",join=true)))
+foreach(rm, filter(endswith(".tex"), readdir("./examples/analysis/tabs/",join=true)))
+
+country = "italy"
+
+include("./analysis_utils.jl")
+include("./error_table_ar.jl")
+include("./error_table_abm.jl")
+include("./error_table_validation_var.jl")
+include("./error_table_validation_abm.jl")
```

Comment:
```suggestion
include("./examples/analysis/tabs/analysis_utils.jl")
include("./examples/analysis/tabs/error_table_ar.jl")
include("./examples/analysis/tabs/error_table_abm.jl")
include("./examples/analysis/tabs/error_table_validation_var.jl")
include("./examples/analysis/tabs/error_table_validation_abm.jl")
```

this way paths are consistent with the rest of the code